webpack.config.js

```diff
@@ -19,17 +19,26 @@ module.exports = {
             { test: /\.css$/, loader: ExtractTextPlugin.extract({
                 fallback: "style-loader",
                 use: "css-loader"
-            }) }
+            }) },
+            { test: /\.(png|jpeg)$/, loader: "url-loader", options: { limit: 8192 } }
         ]
     },
     devtool: "source-map",
     externals: [
-        "mxui/widget/_WidgetBase", "dojo/_base/declare", "dojo/dom-construct", "dojo/dom", "dojo/dom-class",
-        "dojo/dom-style" ],
+        "react",
+        "react-dom",
+        "mxui/widget/_WidgetBase",
```

Comment:
mx and dojo externals could be added with regex

`externals: [ "react", "react-dom", /^mxui\/|^mendix\/|^dojo\/|^dijit\// ],`